docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -31,19 +31,20 @@ async function renderApiDocumentation() {
 	await fs.emptyDir(apiDocsDirectoryPath);
 
 	// Process API reports
+	console.log("Loading API model...");
```

Comment:
Noticed that the console logging here had gotten mixed up a bit while I was debugging something. Figured I might as well fix this too.